src/controllers/User/Account/deleteAccount.js

```diff
@@ -22,11 +37,11 @@ module.exports.deleteAccount = (req, res) => {
             userData.password,
             (err, result) => {
               if (err) handleError500(err);
-              else if (!result) res.status(403).send("Incorrect Password.");
```

Comment:
For better security, avoid revealing to the user which part of the passed data is wrong.

Hope that is clear.
Massiles.